build.sh

```diff
@@ -1,14 +1,22 @@
 #!/bin/bash
 
-rake clobber build
+# Need to remove extra files and build workup gem
+bundle exec rake clobber
 
+# Time to omnibus this
 pushd omnibus
 
+# remove previous traces of builds and install dependencies
 rm -rf pkg
-kitchen test macos
-rm -rf .bundle
-rm -rf vendor
-rm -f Gemfile.lock
-kitchen test windows
+bundle install
```

Comment:
How about bundle install twice then, once in the omnibus dir and once outside of it